TISServices/TISServices/Web.config

```diff
@@ -15,14 +15,15 @@
   </WebServiceSettings>
 
   <connectionStrings>
-    <add name="TDSQC" connectionString="TODO:MyTISDb" providerName="System.Data.SqlClient" />
+    <add name="itembank" connectionString="Data Source=[DATABASE IP OR URL];Initial Catalog=OSS_Itembank;User id=[USERNAME];Password=[PASSWORD]" providerName="System.Data.SqlClient" />
+    <add name="TDSQC" connectionString="Data Source=[DATABASE IP OR URL];Initial Catalog=OSS_TIS;User id=[USERNAME];Password=[PASSWORD]" providerName="System.Data.SqlClient" />
   </connectionStrings>
   
   <appSettings>
-    <add key="LogFilePath" value="D:\MyFileServer\Logs\TISServicesLog.txt"/>
-    <add key="AuthTokenCache:MaxSize" value="100"/>
-    <add key="AuthTokenCache:PurgeCount" value="9"/>
-    <add key="AuthTokenCache:SlidingExpirationMinutes" value="240"/>
+    <add key="LogFilePath" value="[DRIVE]:\path\to\logs\TISServicesLog.txt" />

```

Comment:
Made this more explicit that a user/developer is expected to replace this value.